test/Microsoft.ML.Tests/TrainerEstimators/TreeEstimators.cs

```diff
@@ -308,7 +308,9 @@ public void LightGbmMulticlassEstimatorWithOptions()
         {
             var options = new LightGbmMulticlassTrainer.Options
             {
-                EvaluationMetric = LightGbmMulticlassTrainer.Options.EvaluateMetricType.Default
+                EvaluationMetric = LightGbmMulticlassTrainer.Options.EvaluateMetricType.Default,
+                UseSoftmax = true
```

Comment:
You may want two unit tests total; both setting it explicitly as `true` and `false`. I believe the hyerparameter defaults to `<auto>`, where LightGBM decides for you.

Reply:
Fixed. Thanks!

---
In reply to: [345950381](https://github.com/dotnet/machinelearning/pull/4472#discussion_r345950381) [](ancestors = 345950381)